src/Modal/Modal.js

```diff
@@ -8,11 +8,23 @@ class ModalWindow extends Component {
     static propTypes = {
         modalBody: PropTypes.string.isRequired,
         modalTitle: PropTypes.string.isRequired,
-        isModalOpen: PropTypes.bool,
-        closeModal: PropTypes.func,
-        deletePost: PropTypes.func
+        isModalOpen: PropTypes.bool.isRequired,
+        enableButtons: PropTypes.bool.isRequired,
+        closeFunction: PropTypes.func,
+        confirmFunction: PropTypes.func
     };
 
+    __getButtons(closeFunction, confirmFunction) {
+        if (this.props.enableButtons) {
+            return (
+                <Modal.Footer>
+                    <Button onClick={closeFunction}>Close</Button>
```

Comment:
Make it more universal. What if I would like to use it as alert with one button? Maybe it is good idea to pass configuration instead of predefined values?